sql-plugin/src/main/scala/com/nvidia/spark/rapids/RegexParser.scala

```diff
@@ -441,16 +441,7 @@ class CudfRegexTranspiler(replace: Boolean) {
           // workaround for https://github.com/rapidsai/cudf/issues/9619
           RegexCharacterClass(negated = true, ListBuffer(RegexChar('\r'), RegexChar('\n')))
         case '$' =>
-          RegexSequence(ListBuffer(
-            RegexRepetition(
-              RegexCharacterClass(negated = false,
-                characters = ListBuffer(RegexChar('\r'))),
-              SimpleQuantifier('?')),
-            RegexRepetition(
-              RegexCharacterClass(negated = false,
-                characters = ListBuffer(RegexChar('\n'))),
-              SimpleQuantifier('?')),
-            RegexChar('$')))
+          throw new RegexUnsupportedException("line anchor $ is not supported")
```

Comment:
Should this have a link to the issue as is done below for the same case?

Reply:
Yes, and the other case was actually a bug .. an escaped `$` should have been dealt with as a literal `$` and not an anchor.